src/app/producer-shelf/factories/camera-key-binding.factory.ts

```diff
@@ -0,0 +1,21 @@
+import { KeyBinding } from '../models/key-binding'
+import { Injectable } from '@angular/core'
+
+@Injectable()
+export class CameraKeyBindingFactory {
```

Comment:
Does this need to this specific e.i. does it need to be a `CameraKeyBindingFactory` and not an `ActionKeyBindingFactory`?

Reply:
Are you doing this in your PR @KvelaGorrrrnio?